### ISSUE TITLE
Group posts by Tags/Categories

### ISSUE THREAD

**Issue by jonathanyeong** [jonathanyeong/personal-website-2020#26]
- [ ] Dependent on #3

### Acceptance Criteria
- On the posts/ (writing) page, group posts by categories. 
- On the posts/ page, have a tag or category picker at the top. For initial release, this could be a list of all tags and it will link directly to the page. For future releases, it would be nice to have a dropdown that would filter the page for you.
